spec/cocina/models/builders/title_builder_spec.rb

```diff
@@ -30,7 +31,7 @@
       }
     end
 
-    it 'returns the first title object and logs a deprecation' do
+    it '.build returns the first title as a string and logs a deprecation' do
```

Comment:
This change suggests that what's missing hereabouts is a `describe '.build' do` block containing these `it` tests. But since you are actively working through this class and its methods and have plans for follow-on work, I'm inclined to leave this feedback for later consideration, approve, and merge.